pulp_smash/tests/rpm/api_v2/test_updateinfo.py

```diff
@@ -268,18 +305,13 @@ def setUpClass(cls):
     def test_01_sync_publish(self):
         """Sync and publish the repository.
 
-        Fetch ``updateinfo.xml`` and verify that:
+        When executed, this test method will fetch ``updateinfo.xml`` and
+        verify that:
 
-        * an erratum with id :data:`pulp_smash.constants.RPM_ERRATUM_ID` is
-          present,
-        * the erratum has one "pkglist" child element, and
-        * the "pkglist" element has at least the following child elements:
-
-          .. code-block:: xml
-
-            <collection short="">
-                <name>1</name>
-            </collection>
+        * An ``<update>`` with an ``<id>`` of
+          :data:`pulp_smash.constants.RPM_ERRATUM_ID` is present, and
+        * one of its child ``<package>`` elements has a "name" attribute equal
+          to :data:`pulp_smash.constants.RPM_ERRATUM_RPM_NAME`.
```

Comment:
As mentioned in the PR title, I'm making the test assertions only on "important" information. This change in the docstring reflects the change in focus.